src/Catalyst.Core.Modules.Consensus/ConsensusModule.cs

```diff
@@ -52,12 +57,21 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<TransactionComparerByFeeTimestampAndHash>().As<ITransactionComparer>();
             builder.RegisterType<DeltaHub>().As<IDeltaHub>().SingleInstance();
             builder.RegisterType<DeltaTransactionRetriever>().As<IDeltaTransactionRetriever>().SingleInstance();
-            builder.RegisterType<DeltaBuilder>().As<IDeltaBuilder>().SingleInstance();
             builder.RegisterType<CycleSchedulerProvider>().As<ICycleSchedulerProvider>();
             builder.RegisterType<Consensus>().As<IConsensus>().SingleInstance();
             builder.RegisterInstance(CycleConfiguration.Default).As<ICycleConfiguration>();
 
-            base.Load(builder);
+            builder.RegisterType<DeltaBuilder>().As<IDeltaBuilder>().SingleInstance()
+               .WithExecutionParameters(builder);
+        }
+
+        /// <summary>
+        /// Resolves a parameter of specific type with its named service.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        sealed class ByTypeNamedParameter<T> : ResolvedParameter
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private type 'ByTypeNamedParameter'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)